tmpreaper/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:xenial
+RUN apt-get update -qq && apt-get install -y tmpreaper
+
+FROM alpine:latest
+COPY --from=0 /usr/sbin/tmpreaper /bin/tmpreaper
```

Comment:
sanity check: ldd /bin/tmpreaper shows only the below two shared libs?

Reply:
Also linux-vdso.so.1 - I assumed the alpine kernel provides that since tmpreaper is able to run in the alpine container, but is there a good way to verify that?